linux/ibus-keyman/tests/meson.build

```diff
@@ -67,16 +71,6 @@ test(
   protocol: 'exitcode'
 )
 
-test(
-  'setup-wayland',
-  setup_tests,
-  args: ['--wayland', env_file, pid_file],
-  env: test_env,
-  priority: -20,
-  is_parallel: false,
-  protocol: 'exitcode'
-)
-
 test(
   'teardown-x11',
   teardown_tests,
```

Comment:
Is teardown_tests supposed to occur after all the tests are run? (e.g. towards the end of the file)

Reply:
The order is determined by the priority.